### DISCUSSION TITLE
Cross-validation

### DISCUSSION THREAD

**Discussion by sunjin-k** [snap-stanford/ogb#341] [General]
Hi, I'm trying to do the molecular graph property prediction tasks (not the large scale challenges). 
Can I do cross validation using the official training plus the official validation dataset? (This would involve the official validation dataset being used as input into the model in some of the folds.) Or am I not allowed to use the official validation dataset as input into a model in not just the final submitted model but also the models used as part of the hyperparameter search?

Thanks in advance.

**Comment by weihua916** [comment]
Hi! Good question!
Cross-validation is definitely a good practice, but we dot allow that for the leaderboard submission. See the rules [here](https://ogb.stanford.edu/docs/leader_rules/).

**Reply by sunjin-k** [reply]
Okay, thanks a lot!
